acquisitions/src/main/resources/thunderjet/consortia/features/pieces-api-test-for-cross-tenant-envs.feature

```diff
@@ -0,0 +1,498 @@
+    And match response.titleId == '#(titleId)'
+    And match response.itemId == '#(oldItemId)'
+    And match response.holdingId == '#present'
+    And match response.holdingId != '#(oldHoldingId)'
+    And def holdingId = response.holdingId
+
+    # Check the updated holding record in specified tenant
+    * configure headers = targetHeaders
+    Given path '/holdings-storage/holdings/', holdingId
+    When method GET
+    Then status 200
+    And match response.instanceId == '#present'
+
+    # Check the old holding record as it should not be deleted in specified tenant
+    Given path '/holdings-storage/holdings/', oldHoldingId
+    When method GET
+    Then status 200
+
+    # Check the old item as it should not be deleted in specified tenant
+    Given path '/inventory/items'
+    And param query = 'holdingsRecordId=' + holdingId
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.items[0].holdingsRecordId == '#(holdingId)'
+    And match response.items[0].purchaseOrderLineIdentifier == '#(poLineId)'
+
+
+  Scenario: Check Holding and Item updated in member tenant when updating piece with itemId and deleteHolding=true
+    # Get existing holdingId and itemId in specified tenant
+    Given path 'orders/pieces', pieceId
+    When method GET
+    Then status 200
+    And match response.itemId == '#present'
+    And match response.holdingId == '#present'
+    And def oldItemId = response.itemId
+    And def oldHoldingId = response.holdingId
+
+    # Update piece
+    * set minimalPiece.id = pieceId;
+    * set minimalPiece.titleId = titleId;
+    * set minimalPiece.poLineId = poLineId;
+    * set minimalPiece.itemId = oldItemId;
+    * set minimalPiece.locationId = universityLocationsId;
+    * set minimalPiece.receivingTenantId = targetTenant;
+    Given path 'orders/pieces', pieceId
+    And param createItem = true
+    And param deleteHolding = true
+    And request minimalPiece
+    When method PUT
+    Then status 204
+
+    # Get updated holdingId and itemId in specified tenant
+    Given path 'orders/pieces', pieceId
+    When method GET
+    Then status 200
+    And match response.id == '#(pieceId)'
+    And match response.poLineId == '#(poLineId)'
+    And match response.titleId == '#(titleId)'
+    And match response.itemId == '#(oldItemId)'
+    And match response.holdingId == '#present'
+    And match response.holdingId != '#(oldHoldingId)'
+    And def holdingId = response.holdingId
+
+    # Check the updated holding record in specified tenant
+    * configure headers = targetHeaders
+    Given path '/holdings-storage/holdings/', holdingId
+    When method GET
+    Then status 200
+    And match response.instanceId == '#present'
+
+    # Check the deleted holding record in specified tenant
+    Given path '/holdings-storage/holdings/', oldHoldingId
+    When method GET
+    Then status 404
+
+    # Check the old item in specified tenant
```

Comment:
we use new holdingId, not oldHoldingId, so we should update comment to "Check the new item in specified tenant"
Can also add check that items by oldHoldingId is not exists if possible

Reply:
Will add the check, tough holding cannot be deleted if there are any items.